bin/json-schema-store.js

```diff
@@ -55,4 +55,4 @@ async function refreshSchemas() {
   }
 }
 
-refreshSchemas();
+await refreshSchemas();
```

Comment:
These were all floating promises that began to get flagged because we have `noFloatingPromises` enabled in our common config.

https://github.com/readmeio/standards/blob/92242ec1d1bf08bc1a0c3e4d6f340cdbb74397a6/packages/standards/biome.jsonc#L27-L31